lib/matplotlib/figure.py

```diff
@@ -1564,8 +1564,10 @@ def subfigures(self, nrows=1, ncols=1, squeeze=True,
         wspace, hspace : float, default: None
             The amount of width/height reserved for space between subfigures,
             expressed as a fraction of the average subfigure width/height.
-            If not given, the values will be inferred from a figure or
-            rcParams when necessary.
+            If not given, the values will be inferred from rcParams if using
+            constrained layout (see
+            :class:`~matplotlib.layout_engine.ConstrainedLayoutEngine`), or
+            zero if not using a layout engine.
```

Comment:
We usually don't need `:class:`; I think this should work:
```suggestion
            constrained layout (see `~.ConstrainedLayoutEngine`), or zero if 
            not using a layout engine.
```

Reply:
Indeed it does.  Thanks!